### ISSUE TITLE
IE compatibility

### ISSUE THREAD

**Issue by utilmind** [noerw/leaflet-sidebar-v2#54]
Hello!

The sidebar code contains "Object.assign()" method, which does not supported by IE.

For compatibility with IE, please consider adding the following polyfill:

```
if ("function" !== typeof Object.assign) {
  Object.assign = function(target, varArgs) { // .length of function is 2
    "use strict";
    if (null == target) { // TypeError if undefined or null
      throw new TypeError("Cannot convert undefined or null to object");
    }

    var to = Object(target);

    for (var index = 1; index < arguments.length; ++index) {
      var nextSource = arguments[index];

      if (null != nextSource) { // Skip over if undefined or null
        for (var nextKey in nextSource) {
          // Avoid bugs when hasOwnProperty is shadowed
          if (Object.prototype.hasOwnProperty.call(nextSource, nextKey)) {
            to[nextKey] = nextSource[nextKey];
          }
        }
      }
    }
    return to;
  };
}

```

**Comment by noerw** [comment]
Same issue as #19.
While this polyfill is a nice quickfix, it seems somewhat overkill for our case. I have another fix though.

**Comment by noerw** [comment]
fixed in v3.2.3
